.github/workflows/docs.py

```diff
@@ -0,0 +1,36 @@
+"""Script to generate docs based on given input directory (first argument)
+and output directory (second argument)"""
+import os
+import subprocess
+import sys
+import glob
+import nbformat
+
+code_path = sys.argv[1]
+out_path = sys.argv[2]
+
+os.chdir(code_path)
+os.environ["PDOC_ALLOW_EXEC"] = "1"
+
+for notebook_path in glob.glob("examples/PySDM_examples/*/*.ipynb"):
+    with open(notebook_path, encoding="utf8") as fin:
+        with open(notebook_path + ".badges.md", "w", encoding="utf8") as fout:
+            fout.write(nbformat.read(fin, nbformat.NO_CONVERT).cells[0].source)
+subprocess.run(
+    [
+        "python",
```

Comment:
let's use https://docs.python.org/3/library/sys.html#sys.executable to ensure that the same python executes pdoc as the one executing the script